### PR TITLE
Updates 311 to use TypeScript GraphQL infra

### PR DESCRIPTION
**Pull request by fionawhim**
- Removes the hand-written GraphQL schema in favor of TypeScript types
   that are processed with ts2gql
 - Changes resolvers to typecheck with the Resolvers interface, allowing
   us to remove a ton of explicit type declarations

Also updates some typing in the Open311 service and deletes unused bits
from the Gulpfile.

### CODE REVIEW

**Comment by fionawhim** [comment]
Happy to talk through this in person, and explain `Resolvers` / `ResolvableWith`

**Comment by fionawhim** [comment]
@jessicamarcus Updated to simplify some things.

File: services-js/311/server/graphql/geocoder.ts [hunk 1]
```diff
@@ -1,96 +1,67 @@
...
-  },
+const placeResolvers: Resolvers<Place, Context> = {
+  address: searchResult => searchResult.address,
+  addressId: searchResult => searchResult.addressId,
+  alwaysUseLatLng: searchResult => searchResult.alwaysUseLatLng,
+  exact: searchResult => searchResult.exact,
+  location: searchResult => searchResult.location,
+
+  /**
+   * Looking up units requires an extra call to ArcGIS, so we only do it if
```
Comment (jessicamarcus) [inline services-js/311/server/graphql/geocoder.ts:55]: Thanks for adding these explanatory comments!

**Review by jessicamarcus — APPROVED** [review]
Looks like satisfying spring cleaning! 🌷

**Pull request merged**